NataliaHartgers/readme.md

```diff
@@ -0,0 +1 @@
+# Hello World
```

Comment:
Remove this file (go to this file on your branch on your fork, and click on the trash icon in the top-right)